app/models/concerns/planning_application_decorator.rb

```diff
@@ -35,7 +35,7 @@ def type
     I18n.t(application_type.name, scope: "application_types", default: application_type.description)
```

Comment:
Can this and the relevant yml entries be removed if no longer using?

Reply:
Possibly - will check